src/bin-utils/index.test.js

```diff
@@ -305,6 +311,69 @@ test(
   },
 )
 
+test('specificHelpScript: returns no script available', () => {
+  const config = {scripts: {}}
+  const message = specificHelpScript(config, 'foo')
+  const expected = chalk.yellow('Script foo does not exist')
+  expect(message).toBe(expected)
+})
+
+test(
+  'specificHelpScript: do not display script with flag hiddenFromHelp=true',
+  () => {
+    const config = {
+      scripts: {
+        foo: {
+          description: 'the foo script',
+          script: 'echo "foo"',
+          hiddenFromHelp: true,
+        },
+      },
+    }
+    const message = specificHelpScript(config, 'foo')
+    const expected = chalk.yellow('Script foo does not exist')
+    expect(message).toBe(expected)
+  },
+)
+
+test('specificHelpScript: formats a nice message', () => {
+  const config = {
```

Comment:
Yeah, you know what, I think we should have a single function that takes a config and a prefix and resolves it to:

```
{
  name,
  script,
  description
}
```

That would be really helpful to use in other areas of the package as well. Do you think that you could write that?